src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -554,6 +558,19 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
     });
   }
 
+  attemptDownload(rightClickedFile: any) {
+    let remotePath = rightClickedFile.path;
+    let filename = rightClickedFile.name;
+    let downloadObject = rightClickedFile;
+    let url:string = ZoweZLUX.uriBroker.unixFileUri('contents', remotePath);
+
+    this.downloadService.fetchFileHandler(url,filename, downloadObject).then((res) => {
+                    // TODO: Download queue code for progress bar could go here
+                }).catch((err) => {
```

Comment:
The catch block doesn't do anything aside from rejecting a Promise. The catch bock can be safely removed.